### ISSUE TITLE
Zoom 3D view to cursor instead of center

### ISSUE THREAD

**Issue by andybarry** [rerun-io/rerun#7900]
When using scroll wheel to zoom in a 3D view, I'd like the zoom to move "to the cursor" instead of to the center.  Most other 3D programs do this, like Solidworks, Onshape, etc.  Here's a GIF of Onshape zooming in/out to the cursor.

![onshape-zoom](https://github.com/user-attachments/assets/3f2aa193-ccde-4fa3-a11b-2b4eb63b59dd)

To contrast, this is what Rerun's behavior is:
![rerun-zoom](https://github.com/user-attachments/assets/54e11344-0f5a-438a-a649-0b7bd7f50360)

**Your goals**
The reason this is important is that often folks will use zooming in/out to the cursor to **translate** the view.  I find this method much faster and more intuitive than moving the 3D centerpoint for small translations.

cc @nikolausWest , @gavrelina

**Comment by Wumpf** [comment]
Very related, to other writeups:

* https://github.com/rerun-io/rerun/issues/5451
* https://github.com/rerun-io/rerun/issues/5823

Tl;dr: Depending on where people come from there's wildly different expectations on how camera controls are supposed to behave. The one you point out in the issue is one that we definitely need to have support for (and may or may not consider as the default) as it seems to be common on CAD software. The orbit camera model we have right now is standard in other software for instance Blender.
Note that we also already allow changing to a first person camera model.

But leaving this open because it has nice illustrations and is much more concrete, thanks!
